contracts/Identity.sol

```diff
@@ -10,58 +10,57 @@ contract Identity is KeyManager {
 
   constructor(address owner) public {
     // Add MANAGEMENT_KEY
-    bytes32 key = addressToKey(owner);
-    keys[key].purposes.push(1);
-    keys[key].keyType = 1;
-    keysByPurpose[1].push(key);
+    bytes32 key_ = addressToKey(owner);
+    _keys[key_].purposes.push(1);
+    _keys[key_].keyType = 1;
+    _keysByPurpose[1].push(key_);
   }
 
 
   /**
    * @dev Proxy execution
-   * @param _to address smart contract to call
-   * @param _value uint256 wei supply for proxy execution
-   * @param _data bytes ABI encoded call data
+   * @param to address smart contract to call
+   * @param value uint256 wei supply for proxy execution
+   * @param data bytes ABI encoded call data
    */
   function execute(
-    address _to,
-    uint256 _value,
-    bytes memory _data
+    address to,
+    uint256 value,
+    bytes calldata data
   )
-  public
-  returns (bool success, bytes memory)
+  external
```

Comment:
Is this correct?
- public - all can access
- external - Cannot be accessed internally, only externally.

We changed to external if it is possible because it is cheaper.
For example `addKey` is still public because it is called by `addMultiPurposeKey`.

Reply:
Yes